Document/0x06d-Testing-Data-Storage.md

```diff
@@ -415,7 +415,26 @@ Time: 0.013s
 
 ##### Searching for Cache Databases
 
-iOS applications can store data in cache databases. These databases contain data such as web requests and responses. This data can be sensitive, if tokens, usernames or other information is cached. To display the cache go to the app path and go to `/Library/Caches/{Bundle Identifier}`. The WebKit cache is being stored in the Cache.db file. Objection can interact with it, as it is a normal SQLite database.
+By Default NSURLSession stores data in Cache.db database. This database contain data such as API requests and responses. This database can contain sensitive data, if tokens, usernames or any other sensitive information has been cached. To find the cache files go to the app path and go to `/Library/Caches/{Bundle Identifier}`. The WebKit cache is also being stored in the Cache.db file. Objection can interact with it, as it is a normal SQLite database.
+
+###### Recommendations
+
+It is recommended to disable Caching this data as it may contain sensitive data in the URL parameters or request response. It may cause the violation of the [MSTG-Storage-1](https://github.com/OWASP/owasp-masvs/blob/master/Document/0x07-V2-Data_Storage_and_Privacy_requirements.md "OWASP MASVS").
```

Comment:
If sensitive data is in the URL parameter, it's anyway done wrong :-) Then the data would show up in all log files of all webservers and proxies along the way. So let's concentrate on the request and response in general. 
This text is anyway part of the test case that is referenced to MSTG-Storage-1 and we usually don't reference the requirements again in the text, so I removed it. 

```suggestion
It is recommended to disable Caching this data, as it may contain sensitive information in the request or response. The following list below shows different ways of achieving this: 
```